browse/routes/ui.py

```diff
@@ -183,6 +171,23 @@ def tb_redirect(trackback_id: str, hashed_document_id: str) -> Response:
     raise InternalServerError("Unexpected error")
 
 
+@blueprint.route("tb/<path:arxiv_id>", methods=["GET"])
+def tb(arxiv_id: str) -> Response:
+    """Get trackbacks associated with an article."""
+    response, code, headers = tb_page.get_tb_page(arxiv_id)
+    if code == status.OK:
+        return render_template("tb/tb.html", **response), code, headers  # type: ignore
+    elif code == status.MOVED_PERMANENTLY:
+        return redirect(headers["Location"], code=code)  # type: ignore
+    raise InternalServerError("Unexpected error")
+
+
+@blueprint.route("tb", strict_slashes=False)
```

Comment:
The removed route `@blueprint.route("tb/", defaults={"arxiv_id": ""}, methods=["GET"])` causes flask/werkzeug to redirect /tb to /tb/ and somehow that is ending up a 500.

The route `@blueprint.route("tb", strict_slashes=False)` tells flask/werkz to accept both /tb and /tb/ at this endpoint.